docs/stable/1-client.md

```diff
@@ -94,20 +96,24 @@ To communicate between Status nodes, the [RLPx Transport
 Protocol, v5](https://github.com/ethereum/devp2p/blob/master/rlpx.md) is used, which
 allows for TCP-based communication between nodes.
 
-On top of this we run the RLPx-based subprotocol [Whisper
-v6](https://eips.ethereum.org/EIPS/eip-627) for privacy-preserving messaging.
+On top of this we run the RLPx-based subprotocol [Waku
+V1](https://github.com/vacp2p/specs/blob/master/specs/waku/waku-1.md) for privacy-preserving messaging.
 
 There MUST be a node that is capable of discovering peers and
```

Comment:
This isn't accurate, for v1 of the app we still support Whisper. Instead, it should be MAY for Whisper and SHOULD for Waku. I also think the version in production is v1, but I might be wrong on that.